caluma/caluma_workflow/domain_logic.py

```diff
@@ -259,6 +284,14 @@ def pre_cancel(case, validated_data, user, context=None):
         validated_data["closed_by_user"] = user.username
         validated_data["closed_by_group"] = user.group
 
+        send_event(
+            events.pre_cancel_case,
+            sender="pre_cancel_case",
+            case=case,
+            user=user,
+            context=context,
+        )
```

Comment:
Shouldn't this be below the canceling of ready work items? We should be consistent: either we fire the event before caluma does anything or we fire it after caluma did its pre stuff